.github/workflows/storybook.yml

```diff
@@ -0,0 +1,13 @@
+name: Storybook
```

Comment:
What will this do? Is the idea to make storybook publicly accessible with each deploy? I think that's a neat idea, but what do you think about making this into a separate change?

Reply:
Not about public access but a CI step so that we catch when/if we break Storybook asap.